frontend/src/scenes/insights/BreakdownFilter/taxonomicBreakdownFilterUtils.ts

```diff
@@ -0,0 +1,39 @@
+import { BreakdownType, FilterType } from '~/types'
+import {
+    TaxonomicFilterGroup,
+    TaxonomicFilterGroupType,
+    TaxonomicFilterValue,
+} from 'lib/components/TaxonomicFilter/types'
+import { taxonomicFilterTypeToPropertyFilterType } from 'lib/components/PropertyFilters/utils'
+
+interface FilterChange {
+    multiPropertyBreakdownIsEnabled: string | boolean | undefined
+    breakdownParts: (string | number)[]
+    setFilters: (filters: Partial<FilterType>, mergeFilters?: boolean) => void
+}
+
+export function onFilterChange({ multiPropertyBreakdownIsEnabled, breakdownParts, setFilters }: FilterChange) {
```

Comment:
If you're extracting this then it's a sign that this is doing too much. Business logic should belong to logics, not components.

Reply:
Yep, although it's not the only extracted function in the code :) 

I don't mind one way or another. Was just after a seam to give us a test to avoid the regression we experienced